requirements.txt

```diff
@@ -1,10 +1,10 @@
 # protowhat deps
 markdown2~=2.3.10
 jinja2~=2.11.3
+markupsafe==2.0.1
```

Comment:
Was this previously included in codecov?

Reply:
No It's part of 1 of the packages above but they didn't pin it and the latest version has breaking changes.